doc/RFD900.md

```diff
@@ -1,6 +1,6 @@
 # RFD900
 
-The `RFD900` is a long range radio modem. It works as a transparent serial link. All data that is sent to a modem via `UART` is transmitted to the other modem via RF. A `FTDI` cable can be used to easily connect it to a computer
+The `RFD900` is a long range radio modem. It works as a transparent serial link. All data that is sent to a modem via `UART` is transmitted to the other modem via RF. An `FTDI` cable can be used to easily connect it to a computer

```

Comment:
It should be "A 'FTDI'..." because the word after the a doesnt start with a vocal?